doc/guides/technical-priorities.md

```diff
@@ -0,0 +1,121 @@
+of Node.js in its first 10 years. ESM is the standard that has been adopted as
+the equivalent in the broader JavaScript ecosystem and Node.js must continue to
+develop and improve its ESM implementation in order to stay relevant and ensure
+continued growth for the next 10 years.
+
+* Up to date ES JavaScript Support
+
+JavaScript developers are a fast moving group and need/want support for new ES
+JavaScript features in a timely manner. It is important that Node.js continue
+to provide support for up to date ES versions in order to remain the runtime
+of choice and to ensure its continued growth for the next 10 years.
+
+* Observability
+
+The ability to investigate and resolve problems that occur in applications
+running in production is crucial for organizations. Tools that allow
+people to observe the current and past operation of the application are
+needed to support that need. It is therefore important that the Node.js
+project work towards well understood and defined processes for observing
+the behaviour of Node.js applications as well as ensuring there are well
+supported tools to implement those processes (logging, metrics and tracing).
+This includes support within the Node.js runtime itself (for example
+generating headumps, performance metrics, etc.) as well as support for
+applications on top of the runtime.
+
+* Permissions/policies/security model
+
+Organizations will only choose technologies that allow them to sufficiently
+manage risk in their production deployments. In order for Node.js to
+continue its growth in product/enterprise deployments we need to ensure
+that we help them manage that risk. It is important that we have a well
+documented security model so that consumers understand what threats are/are
+not addressed by the Node.js runtime. We also need to provide
+functions/features which help them limit attack surfaces even if it does
+not result in 100% protection as this will still help organizations
+manage their overall risk level.
+
+* Better multithreaded support
+
+Today servers support multiple threads of concurrent execution.
+It is important that Node.js deployments be able to make full and efficient
+use of the available resources. The right answer is often to use
+technologies like containers to run multiple single threaded Node.js
+instances on the same server. However, there are important use cases
+where a single Node.js instance needs to make use of multiple threads
+in order to achieve a performant and efficient implementation. In addition,
+even when a Node.js instance only needs to consume a single thread to
+complete its work there can be issues. If that work is long running,
+blocking the event loop will interfere with other supporting work like
+metrics gathering and health checks. It is important that the project
+ensures good support for using multiple threads within Node.js
+in order to ensure the continued growth and success of Node.js.
+
+* Single Binary
```

Comment:
This section is a little confusing to me because at first I thought it was referring to the node binary and its library dependencies (e.g. shared vs static) or something. It seems it's actually referring to integrating user code into the node binary itself or something similar.

Reply:
@mscdex correct. Do you have a suggestion for a better title.  I'm thinking but don't have one yet.